src/node_sqlite.cc

```diff
@@ -109,6 +111,12 @@ bool DatabaseSync::Open() {
   int flags = SQLITE_OPEN_READWRITE | SQLITE_OPEN_CREATE;
   int r = sqlite3_open_v2(location_.c_str(), &connection_, flags, nullptr);
   CHECK_ERROR_OR_THROW(env()->isolate(), connection_, r, SQLITE_OK, false);
+  if (allow_load_extension_) {
```

Comment:
If the permission system is enabled, loading extensions should likely be forbidden by default